pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/SingularAboutHealthySelectLogic.java

```diff
@@ -0,0 +1,220 @@
+import com.typesafe.config.ConfigException;
+import com.typesafe.config.ConfigFactory;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.db.DBUtils;
+import org.broadinstitute.ddp.db.dao.FormActivityDao;
+import org.broadinstitute.ddp.db.dao.JdbiActivity;
+import org.broadinstitute.ddp.db.dao.JdbiActivityVersion;
+import org.broadinstitute.ddp.db.dao.JdbiPicklistOption;
+import org.broadinstitute.ddp.db.dao.JdbiUmbrellaStudy;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.broadinstitute.ddp.model.activity.definition.FormActivityDef;
+import org.broadinstitute.ddp.model.activity.definition.question.PicklistQuestionDef;
+import org.broadinstitute.ddp.model.activity.types.ActivityType;
+import org.broadinstitute.ddp.model.activity.types.QuestionType;
+import org.jdbi.v3.core.Handle;
+
+/**
+ * Task for updating the IF_YOU_HAVE_HAD_ARRHYTHMIA_COMPLICATIONS (PATIENT_SURVEY)
+ * and IF_YOU_HAVE_HAD_ARRHYTHMIA (ABOUT_HEALTHY) questions to mark the DO_NOT_REQUIRE
+ * and DID_NOT_REQUIRED options (respectively) as exclusive.
+ * 
+ * <p>See https://broadinstitute.atlassian.net/browse/DDP-8576
+ */
+@Slf4j
+public class SingularAboutHealthySelectLogic implements CustomTask {
+    private static final String PATCH_PATH = "patches";
+    private static final String PATCH_CONF_NAME = "ddp-8576-select-logic.conf";
+    private static final String CONFIG_STUDY_GUID = "study.guid";
+
+    private static class Keys {
+        private static class ChangeSet {
+            private static final String AUTHOR = "author";
+            private static final String DESCRIPTION = "description";
+            private static final String STUDY = "study";
+            private static final String CONTENT = "content";
+        }
+
+        private static class ChangeContent {
+            private static final String ACTIVITY = "activity";
+            private static final String VERSION_TAG = "versionTag";
+            private static final String STABLE_ID = "stableId";
+            private static final String OPTION_ID = "optionId";
+            private static final String EXCLUSIVE = "exclusive";
+        }
+    }
+
+    private Config studyConfig;
+    private Config patchConfig;
+
+    private JdbiUmbrellaStudy studySql;
+    private JdbiActivity activitySql;
+    private JdbiActivityVersion activityVersionSql;
+    private FormActivityDao activityDao;
+    private JdbiPicklistOption picklistOptionSql;
+
+    @Override
+    public void init(Path configPath, Config studyConfig, Config varsConfig) {
+        var studyRoot = configPath.getParent();
+        var patchConfPath = Paths.get(PATCH_PATH, PATCH_CONF_NAME);
+        var patchPath = studyRoot.resolve(patchConfPath);
+
+        final Config patchConfig;
+        try {
+            patchConfig = ConfigFactory.parseFile(patchPath.toFile()).resolveWith(varsConfig);
```

Comment:
`ConfigFactor.parseFile(File)` does not error out of the target file does not exist. Explicitly check for the patch configuration file & bail early if it's not there.